homeassistant/components/eq3btsmart/entity.py

```diff
@@ -1,17 +1,85 @@
 """Base class for all eQ-3 entities."""
 
+from abc import ABC
+
+from homeassistant.core import callback
+from homeassistant.helpers.device_registry import (
+    CONNECTION_BLUETOOTH,
+    DeviceInfo,
+    format_mac,
+)
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
 from homeassistant.helpers.entity import Entity
+from homeassistant.util import slugify
 
 from . import Eq3ConfigEntry
+from .const import (
+    DEVICE_MODEL,
+    MANUFACTURER,
+    SIGNAL_THERMOSTAT_CONNECTED,
+    SIGNAL_THERMOSTAT_DISCONNECTED,
+)
 
 
-class Eq3Entity(Entity):
+class Eq3Entity(Entity, ABC):
```

Comment:
Why the ABC? (I only notice this just now)

Reply:
To signal this class is abstract and can't be instantiated directly. But I can remove it if this is invalid.

Reply:
We usually only add it if we actually have methods that are required to have their own implementation, furthermore we don't use it for this use case in general